src/store/modules/util/UtilState.ts

```diff
@@ -0,0 +1,4 @@
+export default interface UtilState {
+  facilities: any;
```

Comment:
I think we have not used this state anywhere in the code, let me know if I have missed something.